templates/components/products/card.html

```diff
@@ -27,22 +25,34 @@
     {{/if}}>
     <figure class="card-figure">
         {{#if stock_level '===' 0}}
-            {{> components/products/product-badge
-                badge-type='sold-out'
-                badge_view=theme_settings.product_sold_out_badges
-                badge_label=theme_settings.pdp_sold_out_label
-                page_builder_key="pdp_sold_out_label"
-                default_translation=(lang "page_builder.pdp_sold_out_label")
-            }}
-        {{else}}
-            {{#or price.sale_price_with_tax.value price.sale_price_without_tax.value}}
+            {{#if theme_settings.pdp_sold_out_label '===' ''}}
+                {{> components/products/product-badge
+                    badge-type='sold-out'
+                    badge_view=theme_settings.product_sold_out_badges
+                    badge_label=(lang "page_builder.pdp_sold_out_label")
+                }}
+            {{else}}
```

Comment:
can we replace` if statement` inside `badge_label attr` to avoid 2x component usage?

Reply:
Do you know a way how to implement it?

Reply:
Looks like we can only do it inside component. But its up to you